src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java

```diff
@@ -31,6 +36,45 @@ public int doSideEffect() {
   }
 
   public void emptyMethod() {}
+
+  void foo() throws Exception {
+    int a = 90;
+
+    if (a == 1) {
+    } else {} // false-negative until #15664
+
+    if (a == 1) {
+    } else { }
+    // violation above 'Empty blocks should have no spaces. .* may only be represented as {}'
+
+    try (MyResource r = new MyResource()) { }
+    // violation above 'Empty blocks should have no spaces. .* may only be represented as {}'
+    try (MyResource r = new MyResource()) {}
+    try (MyResource r = new MyResource()) {} catch (Exception expected) {} // no warn until #15664
```

Comment:
Please update the comment `// no warn until #15664` to `false-negative until #15664` to be consistent with other such comments in this file.

Reply:
done

Reply:
It was done by me intentionally to avoid extra lines, as line goes above our limit.